src/components/badges/badges.tsx

```diff
@@ -48,12 +43,11 @@ const Wrapper = styled.div`
   height: 2.4rem;
   color: ${({ theme }) => theme.colors.background};
   width: 9.4rem;
-  margin-bottom: 1.5rem;
+`;
 
-  p {
-    font-size: ${({ theme }) => theme.fontSizes.xs};
-    margin-right: 0.3rem;
-  }
+const BadgeLabel = styled.p`
+  font-size: ${({ theme }) => theme.fontSizes.xs};
+  margin-right: 0.3rem;
```

Comment:
Let's put a little warning there, I don't think there should be random `margin-right` ^^

Reply:
I didn't want to make too many changes there (as I thought it was already the case), but after checking a bit of the code base, we have a lot of those. We'll keep improving these. :).

Reply:
Yes, it was really just to put a pin in there for the future

Reply:
Also, would you consider useful for us to switch to a grid system? As much I love it, I'm not sure whether it's worth the time to make the migration (because I see it as expensive)

Reply:
Maybe a bit too soon for that, let's start by rationalizing before that ^^